Essentials/src/com/earth2me/essentials/ReserveEssentials.java

```diff
@@ -0,0 +1,969 @@
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to use for this check.
+     * @param world The name of the {@link World} associated with the amount.
+     * @return True if the account has at least the specified amount of funds, otherwise false.
+     */
+    @Override
+    public boolean hasHoldings(String identifier, BigDecimal amount, String world) {
+        return hasHoldings(identifier, amount);
+    }
+
+    /**
+     * Used to determine if an account has at least an amount of funds.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to use for this check.
+     * @param world The name of the {@link World} associated with the amount.
+     * @return True if the account has at least the specified amount of funds, otherwise false.
+     */
+    @Override
+    public boolean hasHoldings(UUID identifier, BigDecimal amount, String world) {
+        return hasHoldings(identifier, amount);
+    }
+
+    /**
+     * Used to determine if an account has at least an amount of funds.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to use for this check.
+     * @param world The name of the {@link World} associated with the amount.
+     * @param currency The {@link Currency} associated with the balance.
+     * @return True if the account has at least the specified amount of funds, otherwise false.
+     */
+    @Override
+    public boolean hasHoldings(String identifier, BigDecimal amount, String world, String currency) {
+        return hasHoldings(identifier, amount);
+    }
+
+    /**
+     * Used to determine if an account has at least an amount of funds.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to use for this check.
+     * @param world The name of the {@link World} associated with the amount.
+     * @param currency The {@link Currency} associated with the balance.
+     * @return True if the account has at least the specified amount of funds, otherwise false.
+     */
+    @Override
+    public boolean hasHoldings(UUID identifier, BigDecimal amount, String world, String currency) {
+        return hasHoldings(identifier, amount);
+    }
+
+    /**
+     * Used to set the funds to an account.
+     * @param identifier The identifier of the account that is associated with this call.
+     * @param amount The amount you wish to set this accounts's funds to.
+     * @return The {@link EconomyResponse} for this action.
+     */
+    @Override
+    public EconomyResponse setHoldingsDetail(String identifier, BigDecimal amount) {
+        if (!hasAccount(identifier)) return AccountResponse.DOESNT_EXIST;
+        if (!hasAccount(identifier) && !createAccount(identifier)) return AccountResponse.CREATION_FAILED;
```

Comment:
This means an account will never be created. If the account doesn't exist, the method will return before it ever attempts to create an account.

Reply:
Yes nice catch. It completely slipped pass me.